server/models/base.py

```diff
@@ -0,0 +1,22 @@
+import config
+import json
+from abc import ABC, abstractmethod
+
+
+class BaseModel(ABC):
+
+    """
+        This will save the data contained in the inheriting classes 'data' dictionary
+         into a MongoDB collection named after the class that we are saving.
+    """
+    @abstractmethod
+    def get_dic(self):
+        pass
+
+    def save(self, collection_name):

```

Comment:
can we get this to handle `collection.update()` as well if the object already exists in the db

Reply:
I will work on update part in morning.